### PR TITLE
Internal names do not match external names

### PR DESCRIPTION
**Pull request by rhatdan**
We refer to the pause_image and pause_container in the libpod.conf
description, but internally we had infra_image and infra_container.

This meand it the user made changes to the conf, it would not effect the
actual tool using libpod.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Closes:  https://github.com/containers/libpod/issues/2806

**Comment by mheon** [comment]
This breaks existing config files for people actually using them. We should change the manpages if anything.

**Comment by TomSweeneyRedHat** [comment]
LGTM, but would like a head nod from @mheon

**Comment by rhatdan** [comment]
@mheon Well if anyone was using the documented names then they will start working, right now the names are ignored.  

I don't see any reason why a user would have been using the internal names.

**Comment by rhatdan** [comment]
Can we get this merged
@giuseppe @mheon @baude @vrothberg

**Comment by vrothberg** [comment]
No objections from my side.

**Comment by mheon** [comment]
```
infra_image = "k8s.gcr.io/pause:3.1"
infra_command = "/pause"
```

Are in every config file we autogenerate for Rootless - I think we've established the convention already that these are named `infra_`

**Comment by rhatdan** [comment]
@vrothberg can you give it the magic /lgtm

**Comment by vrothberg** [comment]
> @vrothberg can you give it the magic /lgtm

Wanted @mheon's pair of eyes on it that seemed to have caught something here. So it seems that performing this change would allow root to work which was previously broken BUT it will break rootless which is currently working.

IMHO we should change the man page (and the libpod.conf?) to use `infra_` to avoid recursing into a root <-> rootless ping-pong.

@mheon @rhatdan WDYT?

**Comment by haircommander** [comment]
As the person that introduced this inconsistency many months ago, I think using infra* is more accurate

**Comment by rhatdan** [comment]
@haircommander @vrothberg Ok went the other way and also found a crio.conf file we were carrying for no reason so removed it.

File: docs/libpod.conf.5.md [hunk 1]
```diff
@@ -65,13 +65,13 @@ libpod to manage containers.
 **cni_plugin_dir**=""
   Directories where CNI plugin binaries may be located
 
-**pause_image** = ""
-  Pause container image name for pod pause containers.  When running a pod, we
+**infra_image** = ""
+  Ifra (pause) container image name for pod infra containers.  When running a pod, we
```
Comment (TomSweeneyRedHat) [inline docs/libpod.conf.5.md]: s/Ifra/Infra/

File: docs/libpod.conf.5.md [hunk 2]
```diff
@@ -65,13 +65,13 @@ libpod to manage containers.
...
   start a `pause` processes in a container to hold open the namespaces associated with the
```
Comment (TomSweeneyRedHat) [inline docs/libpod.conf.5.md]: Old, but s/processes/process/

File: docs/libpod.conf.5.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/libpod.conf.5.md]: On second thought, might be better to do s/Ifra/A/

Comment (rhatdan) [reply docs/libpod.conf.5.md]: Fixed to say Infra

File: docs/libpod.conf.5.md (hunk 2 above)
Comment (rhatdan) [reply docs/libpod.conf.5.md]: Fixed

**Comment by rhatdan** [comment]
@haircommander @mheon Can I get a /lgtm on this one?

**Pull request merged**